api/test/unit/controllers/test_search_controller.py

```diff
@@ -41,8 +40,7 @@
         (20, 5, 5, (20, 5)),
         # Fewer hits than page size, but result list somehow differs, use that for count
         (48, 20, 50, (20, 0)),
-        # Page count gets truncated always
-        (5000, 10, 10, (5000, MAX_TOTAL_PAGE_COUNT)),
+        (5000, 10, 10, (5000, 5000 / 10)),
```

Comment:
Need a comment here, explaining what this test case is testing for.

Reply:
I believe we can remove this case, this function does not truncate the `page_count` anymore as that logic relies on the serializer now.